VERSION

```diff
@@ -1 +1 @@
-3.0.14
\ No newline at end of file
+3.0.15
```

Comment:
When will you configure your editor to keep file with empty line?

Reply:
I did it... :D